### PR TITLE
signin with uppercase letters

### PR DESCRIPTION
**Pull request by dsims00**
fixes https://github.com/hoodiehq/hoodie/issues/499

### CODE REVIEW

File: tests/integration/routes/session/put-session-test.js [hunk 1]
```diff
@@ -264,3 +264,44 @@ test('PUT /session?include=account.profile', function (group) {
     group.end()
   })
 })
+
+test('PUT /session', function (group) {
```
Comment (gr2m) [inline tests/integration/routes/session/put-session-test.js]: let’s rename this to something like `PUT /session with uppercase letter (hoodiehq/hoodie#499)`

File: tests/integration/routes/session/put-session-test.js [hunk 2]
```diff
@@ -264,3 +264,44 @@ test('PUT /session?include=account.profile', function (group) {
...
+          t.is(response.statusCode, 201, 'returns 201 status')
+          t.deepEqual(response.result.data.id, sessionResponse.data.id, 'returns the right content')
+
+          clock.uninstall()
+          t.end()
+        })
+      })
+      subGroup.end()
+    })
+    group.end()
```
Comment (gr2m) [inline tests/integration/routes/session/put-session-test.js]: we don’t need any grouping, I would replace `group` with `t` and get rid of `subGroup` altogether

Comment (dsims00) [inline tests/integration/routes/session/put-session-test.js]: Ok, fixed.

**Comment by gr2m** [comment]
LGTM

**Comment by HipsterBrown** [comment]
LGTM 🚀 

Great work @dsims00! :clap:

**Pull request merged**